sql/mob_droplist.sql

```diff
@@ -16541,6 +16541,10 @@ INSERT INTO `mob_droplist` VALUES (3138,0,0,1000,1875,1000); -- (Nergal, Bata, A
 INSERT INTO `mob_droplist` VALUES (3139,0,0,1000,1875,1000); -- (Apollyon Demon, Taurus, Ahriman) ancient_beastcoin x 1-2
 INSERT INTO `mob_droplist` VALUES (3139,0,0,1000,1875,500);  -- (Apollyon Demon, Taurus, Ahriman) ancient_beastcoin x 1-2
 INSERT INTO `mob_droplist` VALUES (3140,0,0,1000,2826,350);
+INSERT INTO `mob_droplist` VALUES (3141,0,0,1000,16990,40);
+INSERT INTO `mob_droplist` VALUES (3141,0,0,1000,919,1000);
+INSERT INTO `mob_droplist` VALUES (3141,0,0,1000,1264,1000);
+INSERT INTO `mob_droplist` VALUES (3141,0,0,1000,1237,1000);
```

Comment:
[FFXIDB has these above 100% rate](http://ffxidb.com/zones/153/ancient-goobbue)

But old wiki is capping it at 100%. Anyone feel like killing Acient Goobbue with full TH?

Reply:
That happens when it _can_ drop a 2nd (or 3rd or 4th) of an item, but at least 1 is 100% rate (the additional ones may or may not also be - not required).

So what the person checking this will need to do, is get enough kills to reliably say how many are 100% and how many total there generally could be.